vars/pipelineVegavisorManageNode.groovy

```diff
@@ -358,7 +358,7 @@ void call() {
                             echo "Sleep 30 seconds to ensure epoch have passed"
                             sleep 30
                             script {
-                                def url = "https://api.${env.NET_NAME}.vega.xyz/api/v2/epoch".replaceAll('fairground', 'testnet1')
+                                def url = "https://api-${env.NET_NAME}.vega.rocks/api/v2/epoch".replaceAll('fairground', 'testnet')
```

Comment:
this may not work for other networks than devnet1 now as i see it is for shared for fairground as well